packages/core/src/command/hl7-notification/hl7-notification-webhook-sender-direct.ts

```diff
@@ -62,69 +77,52 @@ export class Hl7NotificationWebhookSenderDirect implements Hl7NotificationWebhoo
       timestampString: sourceTimestamp,
     });
 
-    const bundle = prependPatientToBundle({
+    const newEncounterData = prependPatientToBundle({
       bundle: conversionResult,
       fhirPatient,
     });
     log(`Conversion complete and patient entry added`);
 
-    const nonSpecificUploadParams = {
-      s3Utils: this.s3Utils,
-      contentType: JSON_APP_MIME_TYPE,
-      log,
-      errorConfig: {
-        errorMessage: "Error uploading HL7 FHIR bundle to S3",
-        context: this.context,
-        captureParams: {
-          patientId,
-          cxId,
-          sourceTimestamp,
-        },
-        shouldCapture: true,
-      },
-    };
-
-    // TODO(lucas|ENG-257|2025-05-15): Remove this once new bucket flow is working correctly
-    // Upload to old bucket
-    const oldBundleFilename = buildHl7MessageFhirBundleFileKey({
+    await putAdtConversionBundle({
```

Comment:
nit: this can be execute in parallel to `updateLastestAdtSourcedEncounter`

Reply:
addressed